string-manipulation-3/src/is-palindromic.js

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable no-unused-vars */
+
+function isPalindromic(string) {
+  var subString = string.replace(' ', '');
+  for (var i = 0; i < string.length; i++) {
+    if (subString.length < 2) {
+      return true;
+    }
+    if (subString[0] === subString[subString.length - 1]) {
+      subString = subString.substr(1, subString.length - 2);
+    } else {
+      return false;
+    }
+  }
```

Comment:
You could simplify this a bit by tracking two indexes at once. For example `i` to start at the beginning and `j` to start at the end. Your loop's condition could be that `i < j` and as long as the characters at `i` and `j` are equal, it's a palindrome.

Reply:
Then you wouldn't need to make new substrings every iteration.